core/unix/os_private.h

```diff
@@ -235,6 +236,10 @@ fd_table_add(file_t fd, uint flags);
 
 uint permstr_to_memprot(const char * const perm);
 
+/* The caller needs to bracket this with memquery_iterator_{start,stop}. */
```

Comment:
Can you add to this description to explain what the return value means?

Reply:
Done.